c_emulator/riscv_sim.c

```diff
@@ -136,6 +136,7 @@ static struct option options[] = {
     {"report-arch",                 no_argument,       0, 'a'                     },
     {"test-signature",              required_argument, 0, 'T'                     },
     {"signature-granularity",       required_argument, 0, 'g'                     },
+    {"enable-zpm",                  no_argument,       0, 'Y'                     },
```

Comment:
Please can you use a long option (see the `OPT_...` macros)?

Reply:
Done